tests/serialized_test_data_generation/Makefile

```diff
@@ -0,0 +1,75 @@
+GCR_URL = us.gcr.io/vcm-ml
+REGRESSION_DATA_STORAGE_BUCKET=gs://vcm-fv3gfs-serialized-regression-data
+#<some large conceptual version change>.<serialization statement change>.<hotfix>
+FORTRAN_VERSION=7.1.0
+CWD=$(shell pwd)
+EXPERIMENT ?=c12_6ranks_standard
+
+TEST_DATA_HOST ?=$(CWD)/test_data/$(EXPERIMENT)
+RUNDIR_HOST ?=$(CWD)/rundir/$(EXPERIMENT)
+
+FORTRAN_DIR=$(CWD)/../../
+COMPILED_IMAGE ?= $(GCR_URL)/fv3gfs-compiled:$(FORTRAN_VERSION)-serialize
+
+RUNDIR_CONTAINER=/rundir
+TEST_DATA_CONTAINER=$(RUNDIR_CONTAINER)/test_data
+
+TEST_DATA_TARFILE=dat_files.tar.gz
+TEST_DATA_BUCKET=$(REGRESSION_DATA_STORAGE_BUCKET)/$(FORTRAN_VERSION)/$(EXPERIMENT)
+FV3CONFIG_DIR=$(CWD)/external/fv3config
+LOOKUP_BUCKET=$(shell eval gsutil -q stat $(TEST_DATA_BUCKET)/MetaData-Generator_rank0.json || echo 1)
+
+install_deps:
+	pip3 install -r ../../requirements.txt	
+
+setup_rundir: 
+	if [ ! -d $(RUNDIR_HOST) ]; then $(MAKE) install_deps && python3 make_rundir.py $(EXPERIMENT) ; fi
+
+compile_fortran:
+	cd $(FORTRAN_DIR) && DOCKER_BUILDKIT=1 SERIALIZE_IMAGE=$(COMPILED_IMAGE) $(MAKE) build_serialize_gt4pydev
+
+run_model_make_data:
+	docker run --network host --rm -v $(RUNDIR_HOST):$(RUNDIR_CONTAINER) -v $(TEST_DATA_HOST):$(TEST_DATA_CONTAINER) -it $(COMPILED_IMAGE)-gt4pydev /rundir/submit_job.sh
+
+generate_test_data:
+	if [ ! -d $(RUNDIR_HOST) ]; then $(MAKE) setup_rundir ; fi
+	$(MAKE) compile_fortran
+	$(MAKE) run_model_make_data
+
+pack_test_data:
+	if [ ! -d $(TEST_DATA_HOST)/$(TEST_DATA_TARFILE) ]; then \
+	cd $(TEST_DATA_HOST) && tar  --exclude=input.nml --exclude=*.json  --exclude=$(TEST_DATA_TARFILE) \
```

Comment:
After pack_test_data you can't regenerate it without first calling 'make unpack_test_data' to get the .dat files back, because it removes the dat files. The unpack step will delete the tarfile, you don't have to do it manually. There should either be the tar file, or the .dat files in the directory at a given time. If you run_pack_test_data and don't check for the presence of the tarfile, it will fail because the dat files won't be there.